rascal-vscode-extension/src/lsp/ParameterizedLanguageServer.ts

```diff
@@ -183,6 +183,14 @@ export interface ParserSpecification {
     nonTerminalIsStart?: boolean;
     /** allow ambiguities during parsing, default: false */
     allowAmbiguity?: boolean;
+    /** limit the maximum nesting depth of ambiguities, ambiguities will be pruned below this level. Only of interest if allowAmbiguity or allowRecovery are set to true. default: 2 */
+    maxAmbDepth?: number;
+    /** Allow error recovery during parsing resulting in error trees. default: false */
+    allowRecovery?: boolean;
```

Comment:
Yes 👍🏼 

nit: Let's move `allowRecovery` as the first keyword, and then put a newline before it, to group the recovery config parameters together.

Reply:
I have now added newlines to better show this grouping.